IoTBlockly/ChakraHost/ChakraHost.csproj

```diff
@@ -11,7 +11,7 @@
     <AssemblyName>ChakraHost</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.10586.0</TargetPlatformVersion>
```

Comment:
no more reason than the latest SDK was released, I wanted to test it worked ok...